test/configs/windows2.utsc

```diff
@@ -4,8 +4,8 @@
     "..\\scapy\\contrib\\*.uts"
   ],
   "remove_testfiles": [
-    "test\\cansocket_native.uts",
-    "test\\cansocket_python_can.uts"
+    "cansocket_native.uts",
+    "cansocket_python_can.uts"
```

Comment:
Random fix: windows2.utsc is used from the run_tests.bat files, which are located in a path